docs/introduction/path_to_chrysalis.md

```diff
@@ -24,20 +25,20 @@ The intended outcomes for Chrysalis were:
 
 - **A simpler transition to Coordicide** - With Coordicide making significant progress, we wanted to ensure that all developers and companies that built and deployed on Chrysalis would have as smooth a transition to Coordicide as possible.
 - **Substantial performance improvements** - With the changes introduced by Chrysalis, we saw a substantial improvement in the scalability and reliability of the IOTA Mainnet.
-- **Improved developer and user experience** - The new protocol features, new libraries, and the new wallet madeIOTA one of the best platforms to build on, removing the friction which developers experience today while allowing solutions built on top of the protocol to provide better user experience.
-- **Accelerated adoption** - Chrysalis made IOTA production-ready by becoming a stable protocol, with a reliable set of developer tools and frameworks that will enabled startups, corporations, and governments to develop and launch products powered by IOTA.
+- **Improved developer and user experience** - The new protocol features, new libraries, and the new wallet made IOTA one of the best platforms to build on, removing the friction which developers experience today while allowing solutions built on top of the protocol to provide a better experience for you.
```

Comment:
> one of the best platforms to build on, removing the friction which developers experience today while allowing solutions built on top of the protocol to provide a better experience for you.

Sorry, but what this actually means, how can it be measured and how can it be verified by the reader?

Reply:
I'm not sure I understand the correction/suggestion?

Reply:
IOTA being best is subjective, and friction or quality of an experience is abstract.

Reply:
Hype tweak👍